scripts/macos_build_setup.sh

```diff
@@ -19,11 +23,12 @@ function install_build_dependencies {
 }
 
 function install_qt {
-  echo "Install QT"
-  brew install python@3.10
-  pip3 install -U pip
-  pip3 install aqtinstall
-  aqt install-qt mac desktop ${QT_VERSION} clang_64 -m qtwebengine -O ${QT_INSTALL_DIR}
+  echo "Installing QT ${QT_VERSION}"
+
+  echo "Detected Homebrew prefix: ${BREW_PREFIX}"
+  echo "Qt will be installed to: ${QT_INSTALL_DIR}"
+
+  curl -o qt@5.rb ${QT_FORMULA_URL} && brew install ./qt@5.rb
```

Comment:
Not sure what's the point of this `&&` but we already have `set -e` at the top of this script.
```suggestion
  curl -o qt@5.rb "${QT_FORMULA_URL}"
  brew install ./qt@5.rb
```
Also, quote variables.